go.mod

```diff
@@ -85,7 +85,7 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/gorilla/websocket v1.4.2
 	github.com/k3s-io/helm-controller v0.10.1
-	github.com/k3s-io/kine v0.6.0
+	github.com/k3s-io/kine v0.6.1
```

Comment:
Need to bump to https://github.com/k3s-io/kine/releases/tag/v0.6.2 for the delete fix